lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -101,34 +123,103 @@ def partition_pdf(
         extract_images=False,
         batch_size: int = 1,
         batch_at_a_time=True,
+        local=False,
+        aryn_api_key: str = "",
+        aryn_partitioner_address=DEFAULT_ARYN_PARTITIONER_ADDRESS,
         use_cache=True,
-    ) -> List[List["Element"]]:
-        if batch_at_a_time:
-            return self._partition_pdf_batched(
+    ) -> List[Element]:
+        if not local:
+            return self._partition_remote(
                 file,
+                aryn_api_key,
+                aryn_partitioner_address,
                 threshold,
                 use_ocr,
                 ocr_images,
                 ocr_tables,
                 extract_table_structure,
-                table_structure_extractor,
                 extract_images,
-                batch_size,
-                use_cache,
             )
         else:
-            return self._partition_pdf_sequenced(
-                file,
-                threshold,
-                use_ocr,
-                ocr_images,
-                ocr_tables,
-                extract_table_structure,
-                table_structure_extractor,
-                extract_images,
-                batch_size,
-                use_cache,
-            )
+            if batch_at_a_time:
+                temp = self._partition_pdf_batched(
+                    file,
+                    threshold,
+                    use_ocr,
```

Comment:
nit for future: when passing a bunch of params (specially bools) it's safer to send named arguments, i.e. use_ocr=use_ocr instead of just passing use_ocr. Easy to reorder params accidentily and do the wrong thing